app.js

```diff
@@ -19,11 +20,15 @@ menuBtn.addEventListener('click', () => {
         menuView.classList.remove('slide-out')
         menuView.classList.add('slide-in');
         dim.style.display = 'block';
+        body.classList.remove('no-scroll')
+        body.classList.add('no-scroll')
         state = !state;
     }
     else {
         menuView.classList.add('slide-out');
         menuView.classList.remove('slide-in');
+        body.classList.add('no-scroll')
+        body.classList.remove('no-scroll')
```

Comment:
You added the class list here on `line 30` and removed it in the exact next line on `line 31`
I see something similar on `line 23` and `line 24`
I'm curious how it works, can you share a short screen record of your solution?

Reply:
Made a mistake 
Even tho it works , `line 30` is irrevelant 
Corrected it now


https://user-images.githubusercontent.com/71198309/194110202-a62a0a80-c978-4208-ab26-82d3e73da33b.mp4

Reply:
`line 23` also is not relevant 
removed also 

pardon my ignorance boss :)